### ISSUE TITLE
Fatal error when history in cli is empty

### ISSUE THREAD

**Issue by alpody** [jhunt/hatop#8]
Hatop crashes  after that:

1 . Run ```/hatop -s /var/run/haproxy.stat```
2. Presss ```5``` to enter cli mode
3. Press ```↓``` (down arrow)
4. Script output:
```
Traceback (most recent call last):
  File "./hatop", line 2211, in <module>
    mainloop(screen, opts.interval)
  File "./hatop", line 2103, in mainloop
    next(screen.cli)
TypeError: instance has no next() method
```
I use python 2.7.5 [GCC 4.8.0] on linux2

**Comment by jhunt** [comment]
I was able to reproduce this in the included test lab, on Python 2.7.18.  I'll try to work up a patch this weekend.

**Comment by jhunt** [comment]
Hey @alpody give v0.8.2 a whirl: https://github.com/jhunt/hatop/releases/tag/v0.8.2
